City-of-Pori/pori-kada
# PORI-97: Attraction card list styling

Attraction card list styling without working filters, since they are yet to be implemented.

To test:
1. drush fra -y && drush cc all
2. create attraction cards (kohdekortteja)
3. create attraction listing with same category as the cards
4. create page and add the listing as a widget
5. confirm that the listed items are styled and look good

Note that the filters are not yet working so some coloring of keywords etc will change with the filters and are not shown yet.
